src/services/gquery.js

```diff
@@ -78,9 +78,11 @@ class GQuery {
    */
 
   constructor () {
-    this.apolloClient = createApolloClient(
-      `${window.location.pathname}/graphql`
```

Comment:
This is broken as if the URL visited it `http://localhost:8080/`, `window.location.pathname` would be `/`. This means the client created would use the URL `//graphl`.

Which would try to use `http://graphql` (saw this issue while testing this PR). That's the reason for this change.

Reply:
(and also because if we ever got a URL like `http://cylcuiserver/user/foo/file.html`, the `window.location.pathname` would contain the `file.html`)